app/src/main/java/org/oppia/android/app/devoptions/markchapterscompleted/MarkChaptersCompletedFragmentPresenter.kt

```diff
@@ -78,6 +78,17 @@ class MarkChaptersCompletedFragmentPresenter @Inject constructor(
               )
           }
         }
+      } else if (binding.isAllChecked == true) {
+        binding.isAllChecked = false
+        getMarkChaptersCompletedViewModel().getItemList().forEach { viewModel ->
```

Comment:
Maybe factor this common bit out with the block above to have closer parity with the approach taken for stories and topics (this is a bit more duplicated).

Reply:
Hey @BenHenning please tell me why it is more duplicated?

Reply:
Ah, I was mainly thinking that the forEach bit could be consolidated with the if-check above (``      if (binding.isAllChecked == null || binding.isAllChecked == false) {``), but in retrospect I don't think that would actually reduce much. Please disregard. :)